platform/jvm/capture/build.gradle.kts

```diff
@@ -88,12 +88,31 @@ android {
 cargoNdk {
     librariesNames = arrayListOf("libcapture.so")
     extraCargoBuildArguments = arrayListOf("--package", "capture")
+
+    buildTypes {
+        getByName("release") {
+            buildType = "release"
+            extraCargoBuildArguments = arrayListOf(
+                "--package", "capture",
+                "-Z", "build-std=std,panic_abort",
+                "-Z", "build-std-features=optimize_for_size,panic_immediate_abort",
+            )
+        }
+
+        getByName("debug") {
+            buildType = "dev"
+        }
+    }
+
     module = "../.."
     targetDirectory = "./target"
     // Default set for local dev on ARM-based macos
     targets = arrayListOf("arm64")
     // enable 16 KB ELF alignment on Android to support API 35+
-    extraCargoEnv = mapOf("RUSTFLAGS" to "-C link-args=-Wl,-z,max-page-size=16384")
+    extraCargoEnv = mapOf(
+      "RUSTFLAGS" to "-C link-args=-Wl,-z,max-page-size=16384",
```

Comment:
we should be able to remove this once we update NDK version to 28 https://developer.android.com/guide/practices/page-sizes#compile-r28